### DISCUSSION TITLE
Stylesheet debugger misses extra closing brackets

### DISCUSSION THREAD

**Discussion by twMat** [TiddlyWiki/TiddlyWiki5#6341] [General]
The new [stylesheet debugger](https://tiddlywiki.com/#Debugging%20Stylesheets) doesn't indicate any error when I add an extra `}` to a stylesheet. I understand the debugger is not a *full* bug detector, so an extra closing bracket may just not be a triggers, but I just want to make sure this simple case wasn't *unknowingly* missed out on.

**Comment by pmario** [comment]
Does the extra `}` break the CSS "chain". So are style-sheet tiddlers that come afterwards ignored?

**Reply by twMat** [reply]
> Does the extra `}` break the CSS "chain". So are style-sheet tiddlers that come afterwards ignored?

I don't think so. Is that a condition for the indicator?

**Comment by pmario** [comment]
I did create 2 tiddlers where the second one had an additional closing bracket `}` like so: 

```
.test1 {
  background: red;
}
```


```
.test {
  background: green;
}

}
```

It does break the chain. .. BUT the debugger shows the problem. ... The funny thing is, that if the debugger is shown, both CSS tiddlers work. 
If the debugger isn't shown only class="test1" works. ..

**Reply by twMat** [reply]
I have no idea what to make of that *but* I just want to add that if double brackets are somehow indicated then it should be noted that CSS media queries do use double brackets, e.g
```
@media screen and (min-width: 480px) {
  body {
    background-color: lightgreen;
  }
}
```
They still have to be balanced of course.
